packages/service/pyproject.toml

```diff
@@ -62,14 +62,15 @@ ni-python-styleguide = ">=0.4.1"
 # and regenerate gRPC stubs.
 grpcio-tools = [
   {version = "1.49.1", python = ">=3.8,<3.12"},
-  {version = "1.59.0", python = "^3.12"},
+  {version = "1.59.0", python = ">=3.12,<3.13"},
+  {version = "1.67.0", python = "^3.13"},
```

Comment:
This is incompatible with nimi-python's `protobuf` version constraint: https://github.com/ni/nimi-python/issues/2075

Reply:
nimi-python 1.4.9 fixes this: https://github.com/ni/nimi-python/releases/tag/1.4.9

I'll revisit this after https://github.com/ni/measurement-plugin-python/pull/1076 is merged.